db/migrations/20190603090419_add_name_to_movies.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+exports.up = (Knex) => {
+  return Knex.schema.table('movies', (table) => {
+    table.text('name');
+  })
+  .then(() => {
+    return Knex.raw('ALTER TABLE movies ALTER COLUMN title DROP NOT NULL');
```

Comment:
did you mean to have this as part of another PR: https://github.com/alikslob/sfmovies/pull/13/files ?